docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -17,8 +17,8 @@ Requirements
 To connect to Databricks Delta Lake, you need:
 
 * Tables written by Databricks Runtime 7.3 LTS and 9.1 LTS are supported.
-* Deployments using AWS, HDFS, and Azure Storage are fully supported. Using
-  Google Cloud Storage is not supported.
+* Deployments using AWS, HDFS, and Azure Storage are fully supported. Read
+  operations against Google Cloud Storage (GCS) are also supported.
```

Comment:
Why only read operations? Is it just a lack of integration tests against GCS to give confidence?

Reply:
insert/update/delete operations will fail

Grant reminded me that CTAS and `create table` are supported, I'll clarify that